client/client_test.go

```diff
@@ -121,6 +121,66 @@ func TestListByIndex(t *testing.T) {
 	}
 }
 
+func TestVerifyInclusion(t *testing.T) {
+	env, err := integration.NewLogEnv(context.Background(), 0, "TestVerifyInclusion")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer env.Close()
+	logID, err := env.CreateLog()
+	if err != nil {
+		t.Fatalf("Failed to create log: %v", err)
+	}
+
+	cli := trillian.NewTrillianLogClient(env.ClientConn)
+	client := New(logID, cli, testonly.Hasher, env.PublicKey)
+	// Add a few test leaves.
+	leafData := [][]byte{
+		[]byte("A"),
+		[]byte("B"),
+	}
+
+	if err := addSequencedLeaves(env, client, leafData); err != nil {
+		t.Errorf("Failed to add leaves: %v", err)
+	}
+
+	for _, l := range leafData {
+		if err := client.VerifyInclusion(context.Background(), l); err != nil {
```

Comment:
nit: Use a single context for all tests?

https://github.com/golang/go/wiki/CodeReviewComments#contexts - "Contexts are immutable, so it's fine to pass the same ctx to multiple calls that share the same deadline, cancellation signal, credentials, parent trace, etc."